modules/installation-special-config-crony.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * installing/install_config/installing-customizing.adoc
+
+[id="installation-config-crony_{context}"]
+= Configuring chrony time service
+
+You can set the time server and related settings used by the chrony time service (chronyd)
+by modifying the contents of the `chrony.conf` file and passing those contents
+to your nodes as a MachineConfig.
+
+.Procedure
+
+. Create the contents of the `chrony.conf` file and encode it as base64. For example:
++
+----
+$ cat << EOF | base64
+    server clock.redhat.com iburst
+    driftfile /var/lib/chrony/drift
+    makestep 1.0 3
+    rtcsync
+    logdir /var/log/chrony
+    EOF
+----
+
+. Create the MachineConfig file, replacing the base64 string with the one you create yourself.
+This example adds the file to `master` nodes. You can change it to `worker` or make an
+additional MachineConfig for the `worker` role:
++
+----
+$ cat << EOF > ./masters-chrony-configuration.yaml
+    apiVersion: machineconfiguration.openshift.io/v1
+    kind: MachineConfig
+    metadata:
+    labels:
```

Comment:
Indentation looks off in several of these fields.

Reply:
Overall block indentation (from `apiVersion` line down to `EOF`) affects copy and paste of the code since it doesn't recognize EOF.

Reply:
You were right. I corrected it as follows and tested it with a fresh install. It seems to have worked. Here's what I did. Please verify this:
`
cat << EOF > ./99_masters-chrony-configuration.yaml
apiVersion: machineconfiguration.openshift.io/v1
kind: MachineConfig
metadata:
  labels:
    machineconfiguration.openshift.io/role: master
  name: masters-chrony-configuration
spec:
  config:
    ignition:
      config: {}
      security:
        tls: {}
      timeouts: {}
      version: 2.2.0
    networkd: {}
    passwd: {}
    storage:
      files:
      - contents:
          source: data:text/plain;charset=utf-8;base64,c2VydmVyIGNsb2NrLnJlZGhhdC5jb20gaWJ1cnN0CmRyaWZ0ZmlsZSAvdmFyL2xpYi9jaHJvbnkvZHJpZnQKbWFrZXN0ZXAgMS4wIDMKcnRjc3luYwpsb2dkaXIgL3Zhci9sb2cvY2hyb255Cg==
          verification: {}
        filesystem: root
        mode: 420
        path: /etc/chrony.conf
  osImageURL: ""
EOF

`